doc/build_ES.md

```diff
@@ -87,7 +87,7 @@ Para obtener ayuda sobre los comandos de grin y sus cambios intente:
 
 ```sh
 grin help
-grin wallet --help
+grin server --help
```

Comment:
Note that the spanish build docs are outdated since it doesn't mention `grin wallet` at the end like the others do.